lhotse/cut/multi.py

```diff
@@ -347,6 +354,28 @@ def from_mono(*cuts: Cut) -> "MultiCut":
             }
         )
 
+    def to_mono(self) -> List["DataCut"]:
```

Comment:
```suggestion
    def to_mono(self) -> List["MonoCut"]:
```

Reply:
This would cause a flake8 issue since we cannot import MonoCut (to avoid circular import).